src/dyncfg-ld/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "mz-dyncfg-ld"
```

Comment:
I feel like it's worth fully spelling this out as `mz-dyncfg-launchdarkly`, for consistency with the other (official) `launchdarkly` crates.

Reply:
Done.